app/views/About.js

```diff
@@ -38,9 +37,10 @@ export const AboutScreen = ({ navigation }) => {
       BackHandler.removeEventListener('hardwareBackPress', handleBackPress);
   }, [navigation]);
 
-  const aboutHeaderText = config.tracingStrategy === 'gps'
-    ? t('label.about_header_location')
-    : t('label.about_header_bluetooth');
+  const aboutHeaderText =
+    config.tracingStrategy === 'gps'
```

Comment:
Used a lot, move to `import {isGPS} from 'config';`

Reply:
Good call